app/mender.go

```diff
@@ -682,19 +685,39 @@ func (m *Mender) InventoryRefresh() error {
 		{Name: "mender_client_version", Value: conf.VersionString()},
 	}
 
-	if idata == nil {
-		idata = make(client.InventoryData, 0, len(reqAttr))
+	if iData == nil {
+		iData = make(client.InventoryData, 0, len(reqAttr))
 	}
-	_ = idata.ReplaceAttributes(reqAttr)
+	_ = iData.ReplaceAttributes(reqAttr)
 
-	if idata == nil {
-		log.Infof("No inventory data to submit")
+	if iData == nil {
```

Comment:
Nitpick:
I usually like this in a separate commit, but no hard requirement

Reply:
Generally speaking I agree. But this time I am "using" idata/iData in my commit, so in that sense I think is justified to rename it :)